src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+2. Click your new entity (service) called `getting-started-java`.
+3. Check out the details in the sections for each data type.
+
+### Traces [#traces]
+
+Once you've reached the `getting-started-java` entity in New Relic:
+
+
+1. In the left pane's **Monitor** section, click **Distributed tracing**, and then click the **Fibonacci** trace group.
+    <img
+      alt="Screenshot showing the main distributed tracing page"
+      title="Screenshot showing the main distributed tracing page"
+      src={opentelemetryMainDistributedTracingPage}
+    />
+2. From there, find a trace with an error and click to open it:
+    <img
+      alt="Screenshot showing an error in distributed tracing"
+      title="Screenshot showing an error in distributed tracing"
+      src={opentelemetryDistributedTracingError}
+    />
+3. Once you have the trace open, click `Show in-process spans`, and then click on the resulting span, which will open up a details panel to the right. To see the exception you recorded when a user input is invalid, click on `View span events`:
+    <img
+      title="Screenshot of in-process spans"
+      alt="Screenshot of in-process spans"
+      src={opentelemetryInProcessSpans}
+    />
+
+If you completed the manual instrumentation tutorial, here's how the exception you recorded as a span event will look in New Relic:
+
+<img
+  title="Screenshot showing an exception as a span event"
+  alt="Screenshot showing an exception as a span event"
+  src={opentelemetryExceptiononSpanEvent}
+/>
+
+To view additional details that you set, such as the span attribute, span name, and status code, click on the **Attributes** tab. This pane is also where you can view additional metadata that is automatically collected by the instrumentation libraries you used in this guide, as well as metadata that is attached by New Relic:
+
+<img
+  title="Screenshot of the attributes tab"
+  alt="Screenshot of the attributes tab"
+  src={opentelemetryAttributesTab}
+/>
+
+For more details about viewing your data, see [OpenTelemetry in the New Relic UI](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-distributed-tracing-page)
+
+
+### Metrics [#metrics]
+
+Once you've reached the `getting-started-java` entity in New Relic, you can see a list of all collected metrics, such as runtime metrics (JVMs) and your custom counter attributes.
+
+#### Metrics explorer [#metrics-explorer]
+
+This is a tool that allows you to see a list of your metrics.
+
+1. In the left pane, select **Data > Metrics explorer**, and then select `fibonacci.invocations`.
+
+    <img
+      title="Screenshot showing metrics explorer"
+      alt="Screenshot showing metrics explorer"
+      src={opentelemetryMetricsExplorerMain}
+    />
+2. Under **Dimensions**, view the attributes you collected along with your custom metric, and then click on **fibonacci.valid.n**.
```

Comment:
should this be 'custom metrics'?

Reply:
@zuluecho9 I think it's singular since they just added a single custom counter.